### PR TITLE
Add target for refreshing bugzilla linked PRs

### PR DESCRIPTION
**Pull request by bear-redhat**
Add a Makefile target for refreshing bugzilla-linked PRs.

Ref: Step 5 (After the PR merges, refresh open all master/main PRs with bugzilla/valid-bug) in Code Freeze https://docs.google.com/document/d/19kxmzXFnXrbLChZXBfxy68mCYg06j3qT93VF23wepog

### CODE REVIEW

**Comment by bear-redhat** [comment]
Hold until this target is tested in next code freeze (Jun 24).

**Comment by bear-redhat** [comment]
> 

Thats actually what i am thinking as well... But reading the original PR:
https://github.com/petr-muller/release/commit/bfa9de97c108b3564619fc3107baf704ed82a2b1
It seems the `confirm` was removed from the first job and not added to the second job.

File: Makefile [hunk 1]
```diff
@@ -96,6 +96,14 @@ validate-step-registry:
 	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/ci-operator-configresolver:latest
 	$(CONTAINER_ENGINE) run --platform linux/amd64 --rm -v "$(CURDIR)/core-services/prow/02_config:/prow:z" -v "$(CURDIR)/ci-operator/config:/config:z" -v "$(CURDIR)/ci-operator/step-registry:/step-registry:z" registry.ci.openshift.org/ci/ci-operator-configresolver:latest --config /config --registry /step-registry --prow-config /prow/_config.yaml --validate-only
 
+refresh-bugzilla-prs:
+	mkdir -p /tmp/refresh-bugzilla-prs
```
Comment (smg247) [inline Makefile:100]: can you make this use the `TMPDIR` env var? and then default to `/tmp` if unset?

File: Makefile [hunk 2]
```diff
@@ -96,6 +96,14 @@ validate-step-registry:
...
+	oc get secret --context app.ci github-credentials-openshift-bot -o json | jq -r .data.oauth | base64 --decode > /tmp/refresh-bugzilla-prs/oauth
+	$(CONTAINER_ENGINE) pull gcr.io/k8s-prow/commenter:latest
+	$(CONTAINER_ENGINE) run --platform linux/amd64 --rm -v /tmp/refresh-bugzilla-prs:/etc/oauth:ro gcr.io/k8s-prow/commenter:latest --query="is:pr comments:<2500 state:open base:master label:bugzilla/valid-bug label:lgtm label:approved -label:needs-rebase" --token=/etc/oauth/oauth --updated=0 --ceiling=0 --comment=/bugzilla refresh "The requirements for Bugzilla bugs have changed (BZs linked to PRs on master branch need to target OCP 4.11), recalculating validity."
+	$(CONTAINER_ENGINE) run --platform linux/amd64 --rm -v /tmp/refresh-bugzilla-prs:/etc/oauth:ro gcr.io/k8s-prow/commenter:latest --query="is:pr comments:<2500 state:open base:main label:bugzilla/valid-bug label:lgtm label:approved -label:needs-rebase" --token=/etc/oauth/oauth --updated=0 --ceiling=0 --comment=/bugzilla refresh "The requirements for Bugzilla bugs have changed (BZs linked to PRs on master branch need to target OCP 4.11), recalculating validity."
```
Comment (smg247) [inline Makefile:104]: this could be in a loop over `main` and `master`

**Review by smg247 — COMMENTED** [review]
I think this one is just complex enough that it belongs in a bash script in the `hack` directory. Most of the other targets that aren't just directly using an image do that.

**Review by smg247 — APPROVED** [review]

**Pull request merged**